packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -836,7 +836,8 @@ contract VotingV2 is
 
         // Traverse all requests from the last considered request. For each request see if the voter voted correctly or
         // not. Based on the outcome, attribute the associated slash to the voter.
-        int256 slash = 0;
+        int256 slash = voterStake.unappliedSlash;
+        uint64 requestIndex = voterStake.lastRequestIndexConsidered;
```

Comment:
This variable is also declared in the for loop declaration. I think it could be removed from here.

Reply:
yup!